packages/grafana-data/src/datetime/formatter.test.ts

```diff
@@ -66,18 +66,17 @@ describe('dateTimeFormat', () => {
     });
   });
 
-  const cases = [
-    [1587126975779, { timeZone: 'Africa/Djibouti' }, '2020-04-17 15:36:15'],
-    [1587126975779, { timeZone: 'Europe/London' }, '2020-04-17 13:36:15'],
-    [1587126975779, { timeZone: 'Europe/Berlin' }, '2020-04-17 14:36:15'],
-    [1587126975779, { timeZone: 'Europe/Moscow' }, '2020-04-17 15:36:15'],
-    [1587126975779, { timeZone: 'Europe/Madrid' }, '2020-04-17 14:36:15'],
-    [1587126975779, { timeZone: 'America/New_York' }, '2020-04-17 08:36:15'],
-    [1587126975779, { timeZone: 'America/Chicago' }, '2020-04-17 07:36:15'],
-    [1587126975779, { timeZone: 'America/Denver' }, '2020-04-17 06:36:15'],
-  ];
   describe('when time zone have been set', () => {
-    it.each(cases)('should format with default formatting in correct time zone', (dateInUtc, options, expected) => {
+    it.each([
+      [1587126975779, { timeZone: 'Africa/Djibouti' }, '2020-04-17 15:36:15'],
+      [1587126975779, { timeZone: 'Europe/London' }, '2020-04-17 13:36:15'],
+      [1587126975779, { timeZone: 'Europe/Berlin' }, '2020-04-17 14:36:15'],
+      [1587126975779, { timeZone: 'Europe/Moscow' }, '2020-04-17 15:36:15'],
+      [1587126975779, { timeZone: 'Europe/Madrid' }, '2020-04-17 14:36:15'],
+      [1587126975779, { timeZone: 'America/New_York' }, '2020-04-17 08:36:15'],
+      [1587126975779, { timeZone: 'America/Chicago' }, '2020-04-17 07:36:15'],
+      [1587126975779, { timeZone: 'America/Denver' }, '2020-04-17 06:36:15'],
```

Comment:
You don't have to put the timestamp here, since it is the same for all cases

Reply:
Thank you :)